AzureWebAPI/Services/AzureService.cs

```diff
@@ -0,0 +1,124 @@
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using AzureWebAPI.Models;
+
+namespace AzureWebAPI.Services
+{
+    public class Azure
+    {
+        static AzureParameters azureParameters;
```

Comment:
Follow this coding style:
Private field - _myField;
Public member - MyFunc(), MyProp, MyField;